### ISSUE TITLE
Profiling data for 12_2_0, 12_3_0_pre1, 12_3_0_pre2

### ISSUE THREAD

**Issue by jpata** [jpata/cms-reco-profiling-web#40]
The profiling data for 12_3_0_pre1: 11834.21, 34834.21 is ready.
```
[jpata@lxplus790 cms-reco-profiling]$ ls /eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre1/slc7_amd64_gcc900/*
/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre1/slc7_amd64_gcc900/11834.21:
cmdLog_profiling.sh  step3_circles.json          step3_igprofMEM.1.gz    step3_TimeMemoryInfo.log    step4_igprof_cpu.txt    step4.root                  step5_igprofCPU.gz      step5.log
step1_JobReport.xml  step3_FastTimerService.log  step3_igprofMEM.200.gz  step4_circles.json          step4_igprofMEM.1.gz    step4_TimeMemoryInfo.log    step5_igprof_cpu.txt    step5.root
step1.log            step3_igprofCPU.1.gz        step3_igprofMEM.399.gz  step4_FastTimerService.log  step4_igprofMEM.200.gz  step5_circles.json          step5_igprofMEM.1.gz    step5_TimeMemoryInfo.log
step1.root           step3_igprofCPU.200.gz      step3_igprof_mem.txt    step4_igprofCPU.1.gz        step4_igprofMEM.399.gz  step5_FastTimerService.log  step5_igprofMEM.200.gz
step2_JobReport.xml  step3_igprofCPU.399.gz      step3_JobReport.xml     step4_igprofCPU.200.gz      step4_igprof_mem.txt    step5_igprofCPU.1.gz        step5_igprofMEM.399.gz
step2.log            step3_igprofCPU.gz          step3.log               step4_igprofCPU.399.gz      step4_JobReport.xml     step5_igprofCPU.200.gz      step5_igprof_mem.txt
step2.root           step3_igprof_cpu.txt        step3.root              step4_igprofCPU.gz          step4.log               step5_igprofCPU.399.gz      step5_JobReport.xml

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre1/slc7_amd64_gcc900/136.889:
cmdLog_profiling.sh  step2_JobReport.xml  step2.log  step2.root  step3_JobReport.xml  step3.log  step3.root  step3_TimeMemoryInfo.log  step4_JobReport.xml  step4.log

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre1/slc7_amd64_gcc900/34834.21:
cmdLog_profiling.sh  step2.log                   step3_igprofCPU.50.gz  step3_igprofMEM.50.gz  step3.root                  step4_igprofCPU.50.gz  step4_igprofMEM.50.gz  step4.root
step1_JobReport.xml  step2.root                  step3_igprofCPU.99.gz  step3_igprofMEM.99.gz  step3_TimeMemoryInfo.log    step4_igprofCPU.99.gz  step4_igprofMEM.99.gz  step4_TimeMemoryInfo.log
step1.log            step3_circles.json          step3_igprofCPU.gz     step3_igprof_mem.txt   step4_circles.json          step4_igprofCPU.gz     step4_igprof_mem.txt
step1.root           step3_FastTimerService.log  step3_igprof_cpu.txt   step3_JobReport.xml    step4_FastTimerService.log  step4_igprof_cpu.txt   step4_JobReport.xml
step2_JobReport.xml  step3_igprofCPU.1.gz        step3_igprofMEM.1.gz   step3.log              step4_igprofCPU.1.gz        step4_igprofMEM.1.gz   step4.log

[jpata@lxplus790 cms-reco-profiling]$ ls /eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre1/*/*
/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre1/11834.21/step3:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre1/11834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre1/11834.21/step5:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre1/34834.21/step3:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre1/34834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3
```

**NEW**:
We also have a new 8-core workflow 136.889, for which we are interested in TimeMemoryInfo in step3: /eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre1/slc7_amd64_gcc900/136.889/step3_TimeMemoryInfo.log to produce the following plots:
![Screenshot from 2022-01-03 13-33-42](https://user-images.githubusercontent.com/69717/147925998-bf875666-ebe2-457b-8249-b710e9274669.png)

We are not running igprof for this workflow, so the other comparisons can be omitted. 

cc @xoqhdgh1002

**Comment by jpata** [comment]
Note also the 12_3_0_pre2 is ready to be propagated to the web page:


```
[jpata@lxplus791 cms-reco-profiling-web]$ ls /eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre2/slc7_amd64_gcc10/*
/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre2/slc7_amd64_gcc10/11834.21:
cmdLog_profiling.sh  step3_FastTimerService.log  step3_igprofMEM.399.gz      step4_igprofCPU.1.gz    step4_igprof_mem.txt        step5_igprofCPU.200.gz  step5_JobReport.xml
step1_JobReport.xml  step3_igprofCPU.1.gz        step3_igprof_mem.txt        step4_igprofCPU.200.gz  step4_JobReport.xml         step5_igprofCPU.399.gz  step5.log
step1.log            step3_igprofCPU.200.gz      step3_JobReport.xml         step4_igprofCPU.399.gz  step4.log                   step5_igprofCPU.gz      step5.root
step1.root           step3_igprofCPU.399.gz      step3.log                   step4_igprofCPU.gz      step4.root                  step5_igprof_cpu.txt    step5_TimeMemoryInfo.log
step2_JobReport.xml  step3_igprofCPU.gz          step3.root                  step4_igprof_cpu.txt    step4_TimeMemoryInfo.log    step5_igprofMEM.1.gz
step2.log            step3_igprof_cpu.txt        step3_TimeMemoryInfo.log    step4_igprofMEM.1.gz    step5_circles.json          step5_igprofMEM.200.gz
step2.root           step3_igprofMEM.1.gz        step4_circles.json          step4_igprofMEM.200.gz  step5_FastTimerService.log  step5_igprofMEM.399.gz
step3_circles.json   step3_igprofMEM.200.gz      step4_FastTimerService.log  step4_igprofMEM.399.gz  step5_igprofCPU.1.gz        step5_igprof_mem.txt

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre2/slc7_amd64_gcc10/136.889:
cmdLog_profiling.sh  step2_JobReport.xml  step2.log  step2.root  step3_JobReport.xml  step3.log  step3.root  step3_TimeMemoryInfo.log  step4_JobReport.xml  step4.log

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_3_0_pre2/slc7_amd64_gcc10/34834.21:
cmdLog_profiling.sh  step2.root                  step3_igprofCPU.gz     step3_JobReport.xml         step4_igprofCPU.1.gz   step4_igprofMEM.50.gz  step4_TimeMemoryInfo.log
step1_JobReport.xml  step3_circles.json          step3_igprof_cpu.txt   step3.log                   step4_igprofCPU.50.gz  step4_igprofMEM.99.gz
step1.log            step3_FastTimerService.log  step3_igprofMEM.1.gz   step3.root                  step4_igprofCPU.99.gz  step4_igprof_mem.txt
step1.root           step3_igprofCPU.1.gz        step3_igprofMEM.50.gz  step3_TimeMemoryInfo.log    step4_igprofCPU.gz     step4_JobReport.xml
step2_JobReport.xml  step3_igprofCPU.50.gz       step3_igprofMEM.99.gz  step4_circles.json          step4_igprof_cpu.txt   step4.log
step2.log            step3_igprofCPU.99.gz       step3_igprof_mem.txt   step4_FastTimerService.log  step4_igprofMEM.1.gz   step4.root

[jpata@lxplus791 cms-reco-profiling-web]$ ls /eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre2/*/*
/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre2/11834.21/step3:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre2/11834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre2/11834.21/step5:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre2/34834.21/step3:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_3_0_pre2/34834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3
```

As above, for 136.889, we only care about the TimeMemoryInfo plots.

**Comment by jpata** [comment]
The release 12_2_0 is also ready to be propagated to the webpage.
```
[jpata@lxplus777 ~]$ ls /eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_2_0/slc7_amd64_gcc900/*
/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_2_0/slc7_amd64_gcc900/11834.21:
cmdLog_profiling.sh         step3_igprofCPU.200.gz    step4_circles.json          step4_igprof_mem.txt        step5_igprofCPU.gz
step1_JobReport.xml         step3_igprof_cpu.txt      step4_FastTimerService.log  step4_JobReport.xml         step5_igprof_cpu.txt
step1.log                   step3_igprofMEM.1.gz      step4_igprofCPU.1.gz        step4.log                   step5_igprofMEM.1.gz
step1.root                  step3_igprofMEM.200.gz    step4_igprofCPU.200.gz      step4.root                  step5_igprofMEM.200.gz
step2_JobReport.xml         step3_igprofMEM.399.gz    step4_igprofCPU.399.gz      step4_TimeMemoryInfo.log    step5_igprofMEM.399.gz
step2.log                   step3_igprof_mem.txt      step4_igprofCPU.gz          step5_circles.json          step5_igprof_mem.txt
step2.root                  step3_JobReport.xml       step4_igprof_cpu.txt        step5_FastTimerService.log  step5_JobReport.xml
step3_circles.json          step3.log                 step4_igprofMEM.1.gz        step5_igprofCPU.1.gz        step5.log
step3_FastTimerService.log  step3.root                step4_igprofMEM.200.gz      step5_igprofCPU.200.gz      step5.root
step3_igprofCPU.1.gz        step3_TimeMemoryInfo.log  step4_igprofMEM.399.gz      step5_igprofCPU.399.gz      step5_TimeMemoryInfo.log

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_2_0/slc7_amd64_gcc900/136.889:
cmdLog_profiling.sh  step2.log   step3_JobReport.xml  step3.root                step4_JobReport.xml
step2_JobReport.xml  step2.root  step3.log            step3_TimeMemoryInfo.log  step4.log

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_2_0/slc7_amd64_gcc900/34834.21:
cmdLog_profiling.sh  step3_circles.json          step3_igprofMEM.1.gz   step3_TimeMemoryInfo.log    step4_igprof_cpu.txt   step4.root
step1_JobReport.xml  step3_FastTimerService.log  step3_igprofMEM.50.gz  step4_circles.json          step4_igprofMEM.1.gz   step4_TimeMemoryInfo.log
step1.log            step3_igprofCPU.1.gz        step3_igprofMEM.99.gz  step4_FastTimerService.log  step4_igprofMEM.50.gz
step1.root           step3_igprofCPU.50.gz       step3_igprof_mem.txt   step4_igprofCPU.1.gz        step4_igprofMEM.99.gz
step2_JobReport.xml  step3_igprofCPU.99.gz       step3_JobReport.xml    step4_igprofCPU.50.gz       step4_igprof_mem.txt
step2.log            step3_igprofCPU.gz          step3.log              step4_igprofCPU.99.gz       step4_JobReport.xml
step2.root           step3_igprof_cpu.txt        step3.root             step4_igprofCPU.gz          step4.log

[jpata@lxplus777 ~]$ ls /eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/*
/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/11834.21:
step3  step4  step5

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/34834.21:
step3  step4
[jpata@lxplus777 ~]$ ls /eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/*/*
/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/11834.21/step3:
mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/11834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/11834.21/step5:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/34834.21/step3:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_2_0/34834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3
```

**Comment by jpata** [comment]
@xoqhdgh1002 kind ping on this. 12_3_0_pre3 is now also available. Note that some igprof outputs are broken (see issues), please ignore them.

As discussed, please make an update with the 11834.21 and 34834.21 workflows only, and please also update the scripts in this repository. We need to work towards getting the results faster, now we are again several releases behind.

**Comment by jpata** [comment]
This was merged in #43. Some fixes are requested for the igprof comparisons on 12_3_0_pre4.
